services/app-api/src/postgres/contractAndRates/updateMCCRSID.ts

```diff
@@ -0,0 +1,50 @@
+import { findContractWithHistory } from './findContractWithHistory'
+import { NotFoundError } from '../storeError'
+import type { PrismaClient } from '@prisma/client'
+import type { ContractType } from '../../domain-models/contractAndRates'
+
+type UpdateMCCRSIDArgsType = {
+    contractID: string
+    mccrsID?: string
+}
+
+// Update the MCCRS record number
+async function updateMCCRSID(
```

Comment:
I went back and forth on if this should be `updateContract` or `updateMCCRSID`?

Reply:
I think its good to leave this as a function that just updates the MCCRSID like you have it. Right now `updateContract` at least at postgres level updates the contract revision. When we eventually build that into graphql I imagine it would update the contract revision form data for state user and mayybe update

Reply:
Oh sorry, I posted this question and updated it to `updateContract` before I saw this response. I will change it back